### DISCUSSION TITLE
Hello! Inpainting.

### DISCUSSION THREAD

**Discussion by letthemeatcakes** [SethRobinson/aitools_client#17] [Q&A]
Is it possible or will it be possible to make multiple in-paints in a queue from a big batch of images of the same size using one mask? Like pick a folder and just in paint hundred of images in a row. 
For example I've made a batch of images in Deforum, basically frames of the video. But about 100 of them have 1 element that ruins the whole animation look and I need to get rid of that element and just continue the pattern within the given frames.

**Comment by letthemeatcakes** [comment]
Sorry If it's been already delivered I've just started checking you app and didn't really go in-depth. But I really need to deal with this ruined frames.

**Comment by SethRobinson** [comment]
Hmm, there is nothing built in to batch handle images like that.  Sorry!
